src/compile/nodes/Binding.ts

```diff
@@ -129,14 +129,24 @@ export default class Binding extends Node {
 			updateDom = null;
 		}
 
+		const dependencyArray = [...this.value.dependencies]
+
+		if (dependencyArray.length === 1) {
```

Comment:
I'm not positive that it's needed, but using `quotePropIfNecessary` here seems like a good idea.

Reply:
We aren't quoting `change.${prop}` in most areas of the app because, for the most part, you cannot reference something that needs quotes in expressions in the template. The exception is when shorthands allow it, such as:

```html
<Comp :my-shorthand-binding-var-has-dashes/>
<div class:my-class-name-shorthand-has-dashes></div>
```

So do shorthand bindings come through here? If so, I agree, we need to `quotePropIfNecessary`.

Reply:
These dependency prop strings are the same ones used like `ctx.${prop}` [a bit further down](https://github.com/sveltejs/svelte/pull/1703/files#diff-e09379ddb0bbbc1d22aaee72afeca8caL237), so I felt comfortable not looking for a quote function.